src/styles/style.css

```diff
@@ -5,56 +5,105 @@
   margin: 0;
 }
 
-input {
-  border: 2px solid #000;
-}
-
-button {
-  box-shadow: 2px 2px 2px black;
-}
-
 button:hover {
   cursor: pointer;
-  transform: scale(0.9);
 }
 
 body {
```

Comment:
I love the styling that you selected! Great work!

Reply:
Thank you very much and I'm glad you liked it!